src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -238,17 +238,26 @@ impl Step for Cargotest {
     }
 
     fn make_run(run: RunConfig<'_>) {
-        run.builder.ensure(Cargotest { stage: run.builder.top_stage, host: run.target });
+        // FIXME: support testing stage 0 cargo?
+        assert!(run.builder.top_stage > 0);
```

Comment:
Question: should we just exit here (gracefully) instead of an assert panic? Or rather, declare this case unsupported, as otherwise the cargo test suite would have to work with both possibly-beta rustc and in-tree rustc?